client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -59,84 +56,136 @@ import {
 	isRequestingSites,
 	getSiteAdminUrl,
 } from 'calypso/state/sites/selectors';
-import {
-	getSelectedSite,
-	getSelectedSiteId,
-	getSelectedSiteSlug,
-} from 'calypso/state/ui/selectors';
+import { getSelectedSite, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
 import './style.scss';
 
 /**
  * Module variables
  */
 const SHORT_LIST_LENGTH = 6;
 const SEARCH_RESULTS_LIST_LENGTH = 12;
-const VISIBLE_CATEGORIES = [ 'new', 'popular', 'featured' ];
-const VISIBLE_CATEGORIES_FOR_RECOMMENDATIONS = [ 'new', 'popular' ];
-
-export class PluginsBrowser extends Component {
-	static propTypes = {
-		isRequestingRecommendedPlugins: PropTypes.bool.isRequired,
-		recommendedPlugins: PropTypes.arrayOf( PropTypes.object ),
-		selectedSite: PropTypes.object,
-		trackPageView: PropTypes.bool,
-		hideHeader: PropTypes.bool,
-	};
 
-	static defaultProps = {
-		trackPageViews: true,
-	};
-
-	constructor( props ) {
-		super( props );
-		this.state = {
-			isMobile: isWithinBreakpoint( '<960px' ),
-		};
-	}
+export const PluginsBrowser = ( props ) => {
+	const { trackPageViews = true, category, search } = props;
```

Comment:
Can we destructure this directly in the arrow function above?

Reply:
Also, it seems that there are more props here 
https://github.com/Automattic/wp-calypso/blob/f77a0942fb45b5af4ef3f84351bcd6fc18e60e65/client/my-sites/plugins/main.jsx#L336-L341

Reply:
done 238c00cbf9e68999a6505bdff03db25040d45500